tests/cases/fourslash/moveToFile_refactorAvailable1.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path='fourslash.ts' />
+
+////namespace ns {
+////    /*a*/export function fn() {
+////    }
+////    fn();
+////    /*b*/
+////}
+
+goTo.select("a", "b");
+verify.not.refactorAvailable("Move to file",
+undefined,
```

Comment:
Please label the undefineds and trues with a comment like `/*includeInteractiveRefactors*/ true)`. This is a lint rule enforced in source code, but we can't really lint test content. But it still helps future readers understand what's being passed to this function.

Reply:
on that topic i think we should lint fourslash and fourslash-server tests as they contain code rather than "test case" as a code